assets/js/modules/search-console/dashboard/dashboard-widget.js

```diff
@@ -126,14 +126,22 @@ class GoogleSitekitSearchConsoleDashboardWidget extends Component {
 		const { selectedStats } = this.state;
 
 		const vAxesMap = {
-			0: 'Clicks',
-			1: 'Impressions',
-			2: 'Average CTR',
-			3: 'Average Position',
+			0: __( 'Clicks', 'google-site-kit' ),
```

Comment:
These were never translated, so I know it's not in the AC/IB to do them, but seemed as good a time as any 😄